cms/static/sass/views/_group-configuration.scss

```diff
@@ -273,23 +275,13 @@
 
           // add a group is below with groups styling
           .action-primary {
-            @include blue-button;
-            @include transition(all .15s);
-            @extend %t-action2;
-            @extend %t-strong;
-            display: inline-block;
-            padding: ($baseline/5) $baseline;
-            text-transform: uppercase;
+            @extend %btn-primary-blue;
```

Comment:
This level of clean up can only be communicated through moving picture - http://media.giphy.com/media/XVt7WjqtLxfz2/giphy.gif. Nice job!